handbook/company/communications.md

```diff
@@ -154,31 +143,54 @@ Team members can request reimbursement through [Gusto]([https://app.gusto.com/ex
+- In general, recurring purchases such as subscription services that will continually stretch the spend limit on a cardholder's Brex card should be paid through other means. 
+- For one time purchases where payment via credit card is the most convenient then the card limit will be temporarily increased to accomodate the purchase.  
+
+#### Expense aquittal process
+Fleeties are encourage to [spend company money like it's their own](#spending-company-money), however to ensure that spend is captured correctly, teammates need to take the following steps after spending money:
+- If the spend is under $50, Fleeties can use the memo field in the Brex expense details to provide context on what the spend was for.
+- If the spend is over $50, Fleeties must attach a receipt for the item and add a memo to provide context on what the spend was for.
+- **In addition:** If the spend is for travel, include the event name and year as the prefix, e.g: "DEFCON 2023 - Uber from airport to hotel"
 
 ### Company travel
 
 #### Attending conferences
 When attending a conference or traveling for Fleet, keep the following in mind:
 - $100 allowance per day for your own personal food and beverage. **(There are many good reasons to make exceptions to this guideline, such as dinners with customers.  Before proceeding, please [request approval from the Head of Business Operations](https://fleetdm.com/handbook/business-operations#intake).**
 - Tipping norms vary by culture.  How you tip when representing the company reflects on Fleet's brand.  When traveling in the United States and using your company Brex card, prepare to tip between 18-20% at restaurants.  For rideshare, takeout, delivery, and other situations where tipping comes up, tip between 10-20%.
-- We highly recommend you order a physical Brex card if you do not have one before attending the conference.
+- If you haven't already, order a physical Brex card before attending the conference.
 - The monthly limit on your Brex card may need to be increased temporarily as necessary to accommodate the increased spending associated with the conference, such as travel.  You can [request that here](https://fleetdm.com/handbook/business-operations#intake) by providing the following information:
   - The start and end dates for your trip.
   - The price of your flight (feel free to optimize a direct flight if there is one that is less than double the price of the cheapest non-direct flight).
   - The price of your hotel per night (dry cleaning is allowable if the stay is over 3 days).
   - The price of the admission fees if attending a conference.
-- Please use your personal credit card for movies, mini bars, and entertainment.  These expenses _will not_ be reimbursed.
-
+- Please use your personal credit card for movies, mini bars, and entertainment.  These expenses _will not_ be approved.
```

Comment:
good change- unfortunately, I think we addressed it differently in a separate PR that happened to get reviewed first today.

@JoStableford Since you haven't had a really hairy merge conflict to deal with yet, I am going to leave this one to you for the practice, but please let Sam or I know if you get stuck